std/experimental/allocator/common.d

```diff
@@ -92,13 +92,14 @@ size_t goodAllocSize(A)(auto ref A a, size_t n)
 Returns s rounded up to a multiple of base.
 */
 @safe @nogc nothrow pure
-package size_t roundUpToMultipleOf(size_t s, uint base)
+size_t roundUpToMultipleOf(size_t s, uint base)
```

Comment:
Also, it's much less generic than it could and should be because there wasn't a need. Once we expose it as a public primitive, the requirements change. It probably needs to accept any two unsigned types.